src/App.tsx

```diff
@@ -1,35 +1,54 @@
-import { Route, Routes } from "react-router-dom";
+import { Route, Routes, useNavigate } from "react-router-dom";
 import Home from "./pages/Home/Home";
 import Error from "./pages/Error/Error";
 import "./styles/main.scss";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import SplashPage from "./pages/SplashPage/SplashPage";
 import Login from "./pages/Login/Login";
 import Register from "./pages/Register/Register";
 import Events from "./pages/Events/Events";
 import About from "./pages/About/About";
 import Calendar from "./pages/Calendar/Calendar";
+import { getAuth, onAuthStateChanged } from "firebase/auth";
 import Account from "./pages/Account/Account";
 
 const App = () => {
-  const [userId, setUserId] = useState<string>("");
+  const setUserId = useState<string>("")[1];
```

Comment:
Now that we have the user from the auth, we don't need this function to set a userID. It is already created by the database so we can remove this functionality. If you console log the user on sign in you can see the userid in the console already created.
Remove the setUserId state throughout the project

Reply:
done